ste/mgr-JobPartMgr.go

```diff
@@ -561,9 +561,13 @@ func (jpm *jobPartMgr) createPipelines(ctx context.Context) {
 				CallerID: fmt.Sprintf("JobID=%v, Part#=%d", jpm.Plan().JobID, jpm.Plan().PartNum),
 				Cancel:   jpm.jobMgr.Cancel,
 			}
-
-			sourceCred = common.CreateBlobCredential(ctx, jobState.CredentialInfo.WithType(jobState.S2SSourceCredentialType), credOption)
```

Comment:
So, actual fix here is to reuse the same credential instead of creating a new one for each pipeline?

Reply:
yes